test/README.asciidoc

```diff
@@ -12,12 +12,14 @@ Source structure
 │   └── …
 └── compose
     └── …
+        ├── [rc]         → configuration
         ├── cmd          → command
         ├── [in]         → start file
         ├── [out]        → end file
         ├── [selections] → selection contents
```

Comment:
I think we should rename those files so that they match the variable name (rename state to selections_desc)